### DISCUSSION TITLE
Welcome to Seelen-UI Discussions!

### DISCUSSION THREAD

**Discussion by eythaann** [eythaann/Seelen-UI#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by houssamchiha** [comment]
Hey developer Thank you so much this is the best but please try to fix those issues
1. The start on startup does not work in my windows 10, it asks me for the admin acceptance than it turns off by it self after closing the problem, I added manually to task manager and it takes along time to load.
2. And I can't add icons from the desktop to the dock menu only pin opened apps.

**Comment by eythaann** [comment]
the reason why I don't use the task manager is to prioritize the app being loaded first before any other app using the task scheduler.

**Comment by BenyeDaotzu** [comment]
I used Seelen UI for the first time and really liked it. Unfortunately, the version I downloaded was the one with the system tray removed, which caused me a lot of trouble. Many operations couldn't be performed smoothly, and many applications couldn't be managed effectively. I spent a lot of time trying to solve this problem until I found out that this feature was removed in the version update. I have now downloaded version 2.4.3, and strongly suggest that either an alternative solution be provided or the feature not be removed at all, as it's really hard to use after removal!
